src/test/java/CellTests.java

```diff
@@ -0,0 +1,29 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class CellTests {
+
+    Stone stone = new Stone(Color.BLACK);
+    Cell cell = new Cell(stone);
+
+    @Test
+    void testGettersAfterDefaultConstructor() {
+        Cell cell = new Cell();
+        Assertions.assertFalse(cell.isOccupied());
+        Assertions.assertNull(cell.getStone());
+    }
+
+    @Test
+    void testGettersAfterCustomConstructor() {
+        Assertions.assertTrue(cell.isOccupied());
+        Assertions.assertEquals(cell.getStone(), stone);
+    }
+
+    @Test
+    void testSetStone() {
+        Stone newStone = new Stone(Color.WHITE);
+        cell.putStone(newStone);
+        Assertions.assertEquals(cell.getStone(), newStone);
```

Comment:
Should override equals method in Stone if using assertEquals

Reply:
You are right